### PR TITLE
[16.0][ADD] partner_contact_tag

### PR DESCRIPTION
**Pull request by carmenbianca**
## Odoo task (if applicable)

https://gestion.coopiteasy.be/web#id=10179&action=479&model=project.task&view_type=form&menu_id=536

## Checklist before approval

- [ ] Tests are present (or not needed).
- [ ] Credits/copyright have been changed correctly.
- [ ] Change log snippet is present.
- [ ] (If a new module) Moving this to OCA has been considered.

### CODE REVIEW

**Comment by victor-champonnois** [comment]
Sorry I told you it was categ_id but it's actually category_id. You should also add a widget_many2many, like here https://github.com/odoo/odoo/blob/e3a2cf7e77b844b6d0ac375a4fe224f05227fa32/odoo/addons/base/views/res_partner_views.xml#L228

**Comment by carmenbianca** [comment]
Good catch @victor-champonnois on the widget. It looks much nicer now. :rocket:

**Comment by victor-champonnois** [comment]
@carmenbianca could you ping me when it's redeployed in test ?

File: partner_contact_tag/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    "application": False,
+    "depends": [
+        "base",
+    ],
+    "excludes": [],
+    "data": [
+        "views/res_partner_views.xml",
+    ],
+    "demo": [],
+    "qweb": [],
```
Comment (huguesdk) [inline partner_contact_tag/__manifest__.py]:
```suggestion
    "depends": [
        "base",
    ],
    "data": [
        "views/res_partner_views.xml",
    ],
```
please [avoid empty keys](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#id6).

File: partner_contact_tag/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+{
+    "name": "Partner Contact Tag",
+    "summary": """
+        Display tag in the pop-up form view when adding a contact to a partner.""",
+    "version": "16.0.1.0.0",
+    "category": "Uncategorized",
```
Comment (huguesdk) [inline partner_contact_tag/__manifest__.py]: most modules in [OCA/partner-contact](https://github.com/OCA/partner-contact) seems to use the "Customer Relationship Management" category.

File: partner_contact_tag/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+{
+    "name": "Partner Contact Tag",
+    "summary": """
+        Display tag in the pop-up form view when adding a contact to a partner.""",
```
Comment (huguesdk) [inline partner_contact_tag/__manifest__.py]:
```suggestion
        Display tags in the pop-up form view when adding a contact to a partner""",
```

File: partner_contact_tag/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -0,0 +1 @@
+Display tag in the pop-up form view when adding a contact to a partner.
```
Comment (huguesdk) [inline partner_contact_tag/readme/DESCRIPTION.rst]:
```suggestion
Display tags in the pop-up form view when adding a contact to a partner.
```

File: partner_contact_tag/views/res_partner_views.xml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">partner_contact_tag.view_partner_form</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//notebook/page/field[@name='child_ids']/form/sheet//field[@name='mobile']"
```
Comment (huguesdk) [inline partner_contact_tag/views/res_partner_views.xml]: why not put it at the same place as on the regular form?

**Review by huguesdk — CHANGES_REQUESTED** [review]
i think that this module could be useful for others and should be proposed to [OCA/partner-contact](https://github.com/OCA/partner-contact).

the module name should say that this is a view change only. the first impression for me was that it was adding the concept of partner tags (model and all).

File: partner_contact_tag/views/res_partner_views.xml (hunk 5 above)
Comment (victor-champonnois) [reply partner_contact_tag/views/res_partner_views.xml]: It's already in the regular form, but not on the popup form

Comment (victor-champonnois) [reply partner_contact_tag/views/res_partner_views.xml]: That's why it's a small change, so it's not obvious it has enough value to be proposed  to OCA

Comment (huguesdk) [reply partner_contact_tag/views/res_partner_views.xml]: > why not put it at the same place as on the regular form?

did you miss the “as” in the sentence?

Comment (victor-champonnois) [reply partner_contact_tag/views/res_partner_views.xml]: Ah maybe ^^

**Comment by victor-champonnois** [comment]
@carmenbianca I redeployed in test

File: partner_contact_tag/views/res_partner_views.xml (hunk 5 above)
Comment (carmenbianca) [reply partner_contact_tag/views/res_partner_views.xml]: It's technically, visually, in the same spot as on the regular form—in the bottom right.

**Comment by carmenbianca** [comment]
Implemented fixes suggested by @huguesdk 

Moved to https://github.com/OCA/partner-contact/pull/1489

**Pull request closed without merging**